cli/cliconfig/config.go

```diff
@@ -149,6 +150,72 @@ func SetEnvVarsFromConfigFile(configFile string, orig_env_vars map[string]string
 	return hzn_vars, metadata_vars, nil
 }
 
+// set up the environment variables from the given non-jsonfile.
+// skip the ones that's alrady there originally if override_env is false
+// it returns the hzn env vars from the given file
+func SetEnvVarsFromNonJsonFile(configFile string, orig_env_vars map[string]string, override_env bool) (map[string]string, error) {
+	hzn_vars := map[string]string{}
+
+	if _, err := os.Stat(configFile); err != nil {
+		cliutils.Verbose(fmt.Sprintf("Config file does not exist: %v.", configFile))
+
+		// return no error here because the file does not exists.
+		return hzn_vars, nil
+	}
+
+	cliutils.Verbose("Reading configuration file: %v", configFile)
+
+	// read the configuration from the file
+	hzn_vars, err := GetConfigFromNonJsonFile(configFile)
+	if err != nil {
+		return nil, err
+	}
+
+	// set the env variables
+	if err := SetEnvVars(hzn_vars, orig_env_vars, override_env); err != nil {
+		return hzn_vars, fmt.Errorf("Failed to sent the environment variable defined in file %v. %v", configFile, err)
+	}
+
+	return hzn_vars, nil
+}
+
+func GetConfigFromNonJsonFile(configFile string) (map[string]string, error) {
+	hzn_vars := map[string]string{}
+
+	fileHandle, err := os.Open(configFile)
+	if err != nil {
+		return hzn_vars, err
+	}
+	defer fileHandle.Close()
+
+	// regex for comment line
+	r, _ := regexp.Compile(`^(\s)*#(.*)*`)
+
+	scanner := bufio.NewScanner(fileHandle)
+	for scanner.Scan() {
+		// skip the comment line
+		if r.MatchString(scanner.Text()) {
+			continue
+		}
+
+		// now handle the normal line
+		a := strings.Split(scanner.Text(), "=")
```

Comment:
I think need to remove whitespace, for the case where the contents of the file look like "var = val" instead of "var=val"